.github/workflows/pages.yml

```diff
@@ -0,0 +1,41 @@
+name: Dokka publication
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ macOS-latest ]
```

Comment:
Why macos?

Reply:
Just copied build.yml, then removed Windows

Reply:
It does not matter of course

Reply:
You should probably use linux (it is easier to debug) and remove caches.